kvstore/godoc.go

```diff
@@ -0,0 +1,12 @@
+// Package kvstore provides a simple interface for a key-value store, the
+// `MapStore` and a simple implementation of this interface. These are provided
```

Comment:
Left another comment, but we'll need to update the godoc if we make a submodule for simplemap.